actions/upload_sessions.go

```diff
@@ -215,3 +220,31 @@ func (usr *UploadSessionResource) CreateBeta(c buffalo.Context) error {
 	}
 	return c.Render(200, r.JSON(res))
 }
+
+
+func (usr *UploadSessionResource) GetPaymentStatus(c buffalo.Context) error {
+	session := models.UploadSession{}
+	err := models.DB.Where("id = ?", c.Param("id")).First(&session)
+
+	if err != nil {
+		//TODO: Return better error response when ID does not exist
+		return err
+	}
+
+	paymentStatus := ""
+
+	if session.PaymentStatus == models.PaymentStatusPending {
```

Comment:
Nit: a `switch` statement would be a little cleaner. Also can you make this a method in the model file for `upload_sessions`?